### PR TITLE
LPS-190237 Loading right css files when using RTL languages

### PR DESCRIPTION
**Pull request by antonio-ortega**
Hey,

I'd like to know your opinion about this because when I debugged the issue explained in [LPS-190237](https://liferay.atlassian.net/browse/LPS-190237) I couldn't see those two files where processed by our RTLServlet, what I initially expected.

Not sure whether that's intentional (in which case, my solution might be right) or not (in which case the solution would be to make RTLServlet process these two files)

Thanks!

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
Hmm, this seems odd. I would expect our rtl servlet to already handle this. And if the servlet isn't handling these files for us, I would have expected this to be an issue a long time ago because this code was added over a year ago to request these files.

**Comment by izaera** [comment]
Could this be related to changes made when implementing the CSS client extensions? IIRC, the Echo team changed some things in the ThemeDisplay... :thinking:

**Comment by izaera** [comment]
In any case, I would say this is not intentional and the files should be processed by the RTLServlet...

**Comment by izaera** [comment]
OK, I think that your problem is that the RTLServlet is no longer valid, you should look at [DynamicCSSFilter](https://github.com/liferay/liferay-portal/blob/2abeab7150093bc5fb551ac33c9bf677248b6b09/portal-impl/src/com/liferay/portal/servlet/filters/dynamiccss/DynamicCSSFilter.java) instead.

IINM, long ago, the RTLServlet did runtime transformations to CSS to implement RTL, but we moved that to build time and diverted the file path in `DynamicCSSFilter` to `x.css` or `x_rtl.css` based on the user's language.

In other words: our JARs carry both files inside and it's `DynamicCSSFilter` who decides which one to serve. As opposed to the old schema, where only `x.css` was included in the JAR and the RTLServlet converted it on the fly to `x_rtl.css`.

Am I right?

**Comment by mateusralv** [comment]
ci:test:relevant

**Comment by antonio-ortega** [comment]
Closing this pull request since the actual issue is in build time while generating *_rtl.css file.
